app/models/admin.rb

```diff
@@ -1,18 +1,18 @@
 class Admin < ActiveRecord::Base
   include ActiveModel::ForbiddenAttributesProtection
-  # Authlogic gem
-  acts_as_authentic do |config|
-    config.transition_from_restful_authentication = true
-    if (ArchiveConfig.BCRYPT  || "true")  == "true" then
-      config.crypto_provider = Authlogic::CryptoProviders::BCrypt
-      config.transition_from_crypto_providers = [Authlogic::CryptoProviders::Sha512, Authlogic::CryptoProviders::Sha1]
-    else
-      config.crypto_provider = Authlogic::CryptoProviders::Sha512
-      config.transition_from_crypto_providers = [Authlogic::CryptoProviders::Sha1]
-    end
-  end
+
+  devise :database_authenticatable, \
```

Comment:
I'm pretty certain you don't need all these backslashes as the commas already indicate that the statement isn't finished